masschallenge/django-accelerator
# [AC-8974]: Add user note models

[AC-8974](https://masschallenge.atlassian.net/browse/AC-8974)

How to test:
- Test is passing
- Definition of done matches [AC-8974](https://masschallenge.atlassian.net/browse/AC-8974)
- Check sibling PR https://github.com/masschallenge/accelerate/pull/3016
